maze_transformer/generation/lattice_maze.py

```diff
@@ -11,8 +11,8 @@
 # 	n_nodes: int
 
 SPECIAL_TOKENS: dict[str, str] = dict(
-    adjlist_start="<ADJLIST_START>",
```

Comment:
I'm wary of changing the tokenization scheme, it seems like it will cause a headache for minimal benefit. Everything else in this PR seems fine, but could we leave ADJLIST_START and ADJLIST_END alone?

Reply:
Good call, done